astronomer/providers/databricks/hooks/databricks.py

```diff
@@ -4,15 +4,17 @@
 
 import aiohttp
 from aiohttp import ClientResponseError
+from airflow import __version__
 from airflow.exceptions import AirflowException
 from airflow.providers.databricks.hooks.databricks import (
     GET_RUN_ENDPOINT,
-    USER_AGENT_HEADER,
     DatabricksHook,
     RunState,
 )
 from asgiref.sync import sync_to_async
 
+USER_AGENT_HEADER = {"user-agent": f"airflow-{__version__}"}
```

Comment:
Can you also bump the databricks provider version in the following places 

https://github.com/astronomer/astronomer-providers/blob/943b0cfa1cd4d91eadd734acc6838b85dd21851e/setup.cfg#L58

https://github.com/astronomer/astronomer-providers/blob/943b0cfa1cd4d91eadd734acc6838b85dd21851e/setup.cfg#L110

Reply:
Done